sphinx-docs/screenshot.py

```diff
@@ -203,6 +203,7 @@ class Screenshot(Image):
     option_spec['user-role'] = _parse_user_role
     option_spec['navigation-steps'] = _parse_nav_steps
     option_spec['focus'] = _parse_focus
+    option_spec['registered'] = lambda x: directives.choice(x, ("true", "false"))
```

Comment:
Do you mean are the values "true" and "false" enforced by Sphinx? Yes. It'll throw an error (without affecting other screenshots) if the option is specified incorrectly.